src/helm/benchmark/static/benchmarking.js

```diff
@@ -1045,17 +1045,63 @@ $(function () {
     return $('<td>').append($linkedValue);
   }
 
+  function renderTableHeader(table) {
+    const $tableHeader = $('<thead>');
+    const $row = $('<tr>').append(table.header.map((cell, index) => {
+      $cell = renderCell(cell);
+      // Use the arrow characters to determine the sort order
+      // Ideally the schema would be used instead, but the schema is not available here
+      const sortOrder = cell.value.includes("\u2191") ? "desc" :
+        (cell.value.includes("\u2193") ? "asc" : "");
+      if (sortOrder) {
+        const $sortLink = $("<a>", {"href": "#"}).append("sort").click(() => {
+          $table = $tableHeader.parent('table');
+          $table.find('tbody').remove();
+          $table.append(renderTableBody(table, index, sortOrder));
+          return false;
+        });
+        $cell
+          .append(" [&nbsp;")
+          .append($sortLink)
+          .append("&nbsp;]");
+      }
+      return $cell;
+    }));
+    $tableHeader.append($row);
+    return $tableHeader;
+  }
+
+  function renderTableBody(table, sortColumnIndex, sortOrder) {
+    $tableBody = $("<tbody>");
```

Comment:
Could you add a comment on what the arguments (especially `sortOrder`) are meant to be?

Reply:
Added JSDoc.